can/interfaces/vector/canlib.py

```diff
@@ -657,7 +655,7 @@ def _detect_available_configs() -> List[AutoDetectedConfig]:
                     "vector_channel_config": channel_config,
                 }
             )
-        return configs
+        return configs  # type: ignore
```

Comment:
Could we instead specify `configs: List[AutoDetectedConfig] = []` in the beginning of the method? If not, this is fine.

Reply:
I tried, but TypedDict does not allow extra keys.

Reply:
Okay, apparently this is not supported. Didn't know that.